source/shared/cpp/ObjectModel/BaseInputElement.cpp

```diff
@@ -29,7 +29,5 @@ Json::Value BaseInputElement::SerializeToJsonValue() const
 {
     Json::Value root = BaseCardElement::SerializeToJsonValue();
 
-    root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::IsRequired)] = m_isRequired;
```

Comment:
So, the latest thinking on this, as of, i think yesterday, is that this property will be part of the input validation story. So given that, i'm inclined not to remove it. We've shipped it with every release, and we're going to actually hook it up to something for 1.3, so i don't think there's any reason to pull it (although it should not serialize if it's false).

Reply:
[https://github.com/microsoft/AdaptiveCards/issues/3081](https://github.com/microsoft/AdaptiveCards/issues/3081)

---
In reply to: [300051174](https://github.com/microsoft/AdaptiveCards/pull/3131#discussion_r300051174) [](ancestors = 300051174)

Reply:
@RebeccaAnne / @almedina-ms  does this mean we can abandon this PR/ link and close out #3072 ? If so, then let's do that!

Reply:
I think the better answer should be to add a validation for 

if(!m_isRequired){
Do serialization
}

is that right, @RebeccaAnne ??

Reply:
Yep, that is the correct behavior, and is something that needs to happen either for this bug or as part of the Input Validation work. 

---
In reply to: [300798123](https://github.com/microsoft/AdaptiveCards/pull/3131#discussion_r300798123) [](ancestors = 300798123)